src/main/java/org/symphonyoss/symphony/messageml/elements/Code.java

```diff
@@ -30,29 +39,92 @@
 public class Code extends Element {
 
   public static final String MESSAGEML_TAG = "code";
+  private static final String MML_LANGUAGE_ATTR = "language";
+  private static final String PML_LANGUAGE_ATTR = "data-language";
   public static final char MARKDOWN_DELIMITER_CHAR = '`';
   public static final int MARKDOWN_DELIMITER_LENGTH = 3;
   private static final int MARKDOWN_DELIMITER_INDENT = 0;
 
+  private static final List<String> SUPPORTED_LANGUAGES = Arrays.asList(
+      "plaintext", "c", "cpp", "csharp", "css", "html", "java", "js", "jsx", "php", "python", "r", "typescript", "tsx"
```

Comment:
did not we say we would keep this free so that new languages can be added client side only?